encoder_pub_rate: 20
cmd_pub_rate: 20

motor_count: 1

motor_1: {
  id: 1,
  op_mode: 4,
  low_limit: -4096,
  high_limit: 4096,
  reset_state: 1024,
  gear_ratio: 1,
  offset: 0
}